### PR TITLE
Added Infix to Postfix using stack in python

### PR DESCRIPTION
**Pull request by jankivaghasiya**
### Added infix_to_postfix_using_stack.py in code/python directory
Added Algorithm
Added Test Cases
Added Time and Space Complexity

### CODE REVIEW

**Comment by raksha009** [comment]
Pl link your code with appropriate folder.

File: Code/Python/infix_to_postfix_using_stack.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+Output - ABC*+DE/H*-
+
+Input - (A+B)*C^D^E/(F-G)*H
+Output - AB+CDE^^*FG-/H* 
+         
+Time Complexity: O(n)
+since the time for push and pop operations are considered constant
+
+Space Complexity: O(n)
+'''
```
Comment (raksha009) [inline Code/Python/infix_to_postfix_using_stack.py:109]: Specify n and add a new line at the end

**Review by raksha009 — CHANGES_REQUESTED** [review]

**Comment by jankivaghasiya** [comment]
> Pl link your code with appropriate folder.

I linked it according to the code guidelines. Should I link it under Stack section?

**Comment by raksha009** [comment]
> > Pl link your code with appropriate folder.
> 
> I linked it according to the code guidelines. Should I link it under Stack section?

Yess

File: Code/Python/infix_to_postfix_using_stack.py (hunk 1 above)
Comment (jankivaghasiya) [reply Code/Python/infix_to_postfix_using_stack.py:109]: > Specify n and add a new line at the end

Newline character n?

**Comment by jankivaghasiya** [comment]
@raksha009 mam, I've linked the code under the stack directory and added a new line. Can you please review it.

**Comment by jankivaghasiya** [comment]
@raksha009 mam, I've linked the code under the stack directory and added a new line. Can you please review it?

File: Code/Python/infix_to_postfix_using_stack.py (hunk 1 above)
Comment (plazzy99) [reply Code/Python/infix_to_postfix_using_stack.py:109]: yes

**Review by plazzy99 — CHANGES_REQUESTED** [review]

**Comment by plazzy99** [comment]
Why the code have been inserted at 2 locations

**Pull request closed without merging**